### DISCUSSION TITLE
Airflow nodes taking over 400% CPU usage

### DISCUSSION THREAD

**Discussion by hugh-nguyen** [apache/airflow#22892] [General]
Hi,

I recently changed jobs, I just setup Airflow on my new computer which happens to have the exact same specs and found that it's insanely slow. 

I ran docker stats and found the CPU usage is huge. Some of the nodes reach over 400% at times.

![image](https://user-images.githubusercontent.com/30641101/162648283-411ba18c-543f-4648-94f0-4e09e763f8c1.png)

And this is after dedicating the maximum amount of CPU to my docker daemon.

![image](https://user-images.githubusercontent.com/30641101/162648319-f478dd32-00e3-4686-9ff1-f193e9ef378b.png)

Anyone know what's going on?

**Comment by potiuk** [comment]
Maybe they just do a lot of stuff. Why do you think it's too much ? What do the logs show for those components?
